command/agent.go

```diff
@@ -878,9 +878,10 @@ func (c *AgentCommand) Run(args []string) int {
 func verifyRequestHeader(handler http.Handler) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		if val, ok := r.Header[consts.RequestHeaderName]; !ok || len(val) != 1 || val[0] != "true" {
-			logical.RespondError(w,
-				http.StatusPreconditionFailed,
-				errors.New(fmt.Sprintf("missing '%s' header", consts.RequestHeaderName)))
+			err := errors.New(fmt.Sprintf("missing '%s' header", consts.RequestHeaderName))
+			status := http.StatusPreconditionFailed
+			logical.AdjustErrorStatusCode(&status, err)
+			logical.RespondError(w,	status,	err)
```

Comment:
```suggestion
			logical.RespondError(w, status, err)
```

strange, seems like something the ide should have done

Reply:
hmm, I see the spaces in my ide. not sure what has happened! :)